index.js

```diff
@@ -0,0 +1,116 @@
+let rounds = 1;
+let homeTotalScore = 0;
+let guestTotalScore = 0;
+let matchDuration = 12;
```

Comment:
- [ ] These should come after all selected elements ie  after `line 26`

Reply:
changed